.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Offboarding
+about: An offboarding checklist for folks leaving the team
+title: Offboarding checklist for {GitHub Username}
+labels: ''
+assignees: ''
+---
+
+- [ ] Has been removed from [the FAC-team Google Group](https://groups.google.com/a/gsa.gov/g/fac-team/members)
+- [ ] Has been removed from [the FAC team in GitHub](https://github.com/orgs/GSA-TTS/teams/fac-team/members)
+- [ ] Has been from the private channel #oros-fac-private in Slack (“/remove @[person]”)
```

Comment:
@MelissaBraxton does it make sense to remove people from Figma or other tools if they leave?